core/backend/src/Entity.ts

```diff
@@ -92,6 +108,25 @@ export class Entity {
   /** Get the full BIS class name of this Entity in the form "schema:class". */
   public get classFullName(): string { return this._ctor.classFullName; }
 
+  public get schemaItemKey(): SchemaItemKey { return this._ctor.schemaItemKey; }
```

Comment:
lets tag everything we're adding as @beta during development so we can go through and review everything one last time before we make 5.0 final

Reply:
Done